dags/atd_signal_comms.py

```diff
@@ -155,7 +168,7 @@ def get_start_date(**context):
         command=f"python atd-signal-comms/socrata_pub.py detector --start {start_date} -v --env {DEPLOYMENT_ENVIRONMENT}",
         environment=env_vars,
         tty=True,
-        force_pull=True,
+        force_pull=False,
```

Comment:
These extra `force_pulls` are unnecessary after the first one, I believe.

Reply:
Chia prompted me to adopt this pattern in a PR i have open Charlie - this is super smart. Thanks for this!